release-checks.sh

```diff
@@ -25,3 +26,13 @@ then
     echo "The release process requires a valid GitHub token. See RELEASE.md for details."
     exit 1
 fi
+
+if [ "$(uname)" == "Darwin" ] && [ -z "${GPG_TTY+set}" ]
```

Comment:
This works fine, but since this is a bash script it's much preferable to stick to the more modern `[[` `]]`construct which also provides more predictable results (more details [here](http://mywiki.wooledge.org/BashFAQ/031)). It's also consistent with what we already have in scripts.

So this would become:

`[[ $(uname) == "Darwin" && -z "${GPG_TTY+set}" ]]`

Reply:
Thanks! I changed it to [[ ]].